.github/workflows/ci.yml

```diff
@@ -40,18 +40,20 @@ jobs:
                 args: analyse
 
     ci:
-        name: Test PHP ${{ matrix.php-version }} ${{ matrix.name }}
+        name: Test PHP ${{ matrix.php-version }} & Symfony ${{ matrix.symfony-version }}
         runs-on: ubuntu-latest
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony-version }}
         strategy:
             fail-fast: false
             matrix:
                 php-version: ['7.4', '8.0']
+                symfony-version: ['4.4.*', '5.3.*']
                 composer-flags: ['']
                 name: ['']
                 include:
-                    - php-version: 7.4
-                      composer-flags: '--prefer-lowest'
-                      name: '(prefer lowest dependencies)'
+                    - php-version: '8.0'
+                      symfony-version: '6.0.*'
```

Comment:
It looks like you don't use it

Reply:
It is used, it's an extra run as I can't put Symfony 6.0 in the main matrix as it supports only PHP8.0+

Reply:
you are right, sorry, I missed the line where the export